### ISSUE TITLE
No Python

### ISSUE THREAD

**Issue by NickyBoi13** [AlizerUncaged/desktop-waifu#215]
When i try to run the start.bat file it loads but then says: No Python at 'C:\Python10\python.exe'. any suggestions?

**Comment by carlangassnake** [comment]
You have to install python there, in your C disk in a folder named Python10

**Comment by NickyBoi13** [comment]
can't i just change the python version to python11?

**Comment by carlangassnake** [comment]
Well, as long as you install it there (or change the route where the exe is trying to find Python, which you can do and it's explained in some other post here) I assume there wont be any trouble with other version

**Comment by KasumiChanXx** [comment]
so question on that is that im making a folder named Python10 or is there a folder already made

**Comment by carlangassnake** [comment]
no, you have to create the folder, so the installation route must be this "C:\Python10". Inside that python10 folder, you should target the instalation.
